assets/css/style.css

```diff
@@ -1041,6 +1042,7 @@ a.btn {
   padding: 0.5rem;
   height: 1.5rem;
   width: 2.5rem;
+  height: 2.5rem;
```

Comment:
@lxndrblz was this added by mistake? Now there are two height attributes

Reply:
No, that was by mistake. I forgot to remove the lower value.

Reply:
Hmm, then should we find a way to vertically center the "!" mark in the circle?

Reply:
It should be resolved by setting the height identical to the width. Or am I am missing something?

Reply:
Actually never mind, I thought it wasn't centered vertically in the screenshot but it is. I wonder if I only saw it on my machine but I remember the "!" symbol was not properly vertically centered in the circle.